### PR TITLE
Reactive armour code improvements

### PR DESCRIPTION
**Pull request by TheBonded**
## About The Pull Request

Replaces some variables such as `I` and `A` with `weapon` and `anomaly`

Also replaces every instance of `<span class="type">message</span>` with `span_type("whatever")`

## Why It's Good For The Game

Makes reading the code better, I guess? Also taught me how to code better so that's nice. 

## Changelog

Not player facing

### CODE REVIEW

**Comment by TheBonded** [comment]
I fucked it up

**Comment by SuperNovaa41** [comment]
This is why you don't web edit..

**Comment by TheBonded** [comment]
don't worry I downloaded vscode this will be my last webedit

**Comment by Sealed101** [comment]
undocumented u's everywhere smh

**Comment by TheBonded** [comment]
at least it didn't fail the checks this time

File: code/modules/clothing/suits/reactive_armour.dm [hunk 1]
```diff
@@ -5,47 +5,48 @@
...
 		new armour_path(get_turf(src))
 		qdel(src)
-		qdel(A)
+		qdel(anomaly)
 
-//Reactive armor
+//Reactive armour
 /obj/item/clothing/suit/armor/reactive
-	name = "reactive armor"
+	name = "reactive armour"
```
Comment (MrMelbert) [inline code/modules/clothing/suits/reactive_armour.dm:28]: https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#things-that-do-not-matter

There's zero reason to change all of this it from `armor` to `armour`

```
The following coding styles are not only not enforced at all, but are generally frowned upon to change for little to no reason:

    - English/British spelling on var/proc names 
```

File: code/modules/clothing/suits/reactive_armour.dm [hunk 2]
```diff
@@ -5,47 +5,48 @@
...
-	///Message sent when the armor is still on cooldown, but activates.
-	var/cooldown_message = "<span class='danger'>The reactive armor fails to do much, as it is recharging! From what? Only the reactive armor knows.</span>"
-	///Duration of the cooldown specific to reactive armor for when it can activate again.
-	var/reactivearmor_cooldown_duration = 10 SECONDS
-	///The cooldown itself of the reactive armor for when it can activate again.
-	var/reactivearmor_cooldown = 0
 	icon_state = "reactiveoff"
 	inhand_icon_state = "reactiveoff"
-	blood_overlay_type = "armor"
+	blood_overlay_type = "armour"
```
Comment (MrMelbert) [inline code/modules/clothing/suits/reactive_armour.dm:32]: ...especially cause some of these will break things

**Review by MrMelbert — CHANGES_REQUESTED** [review]

**Comment by TheBonded** [comment]
Yeah... Fair enough. I have another branch that does everything this branch does /without/ those changes for this exact reason; I figured this might happen but thought it was worth a shot. I'll reopen the pr with that.

**Pull request closed without merging**